orbit/pkg/execuser/execuser_windows.go

```diff
@@ -230,3 +238,62 @@ func startProcessAsCurrentUser(appPath, cmdLine, workDir string) error {
 
 	return nil
 }
+
+// setupSyncChannel creates the synchronization channel through which child process will be
```

Comment:
Appreciate the detailed docs in the method!

Reply:
thanks!